test/radcli/cli.go

```diff
@@ -179,18 +141,6 @@ func (cli *CLI) ApplicationShow(ctx context.Context, applicationName string) (st
 	return cli.RunCommand(ctx, args)
 }
 
-// ApplicationList runs a command to list applications and returns the output as a string. It returns an
-// error if the command fails.
-func (cli *CLI) ApplicationList(ctx context.Context) (string, error) {
```

Comment:
Are these covered elsewhere in the tests?

Reply:
These functions are not being referenced anywhere in the code. I am not sure if it is wise to keep these just in case if we need them one day. I believe that if we need, we can add them in future. As of now, it is showing up as `deadcode` when I run `staticcheck`. That is why I believe it is good to remove them.

Reply:
> I was curious why don't we just use these functions to add test coverage unless they are being tested through other code paths, but doesn't look like we are testing these commands. I don't think it is in scope of this PR to add tests, but I have created this issue #7453, we can reference this PR if these functions need to be brought back.
> 
> Thanks for doing this cleanup!

Our coverage results don't include functional tests. To do that we'd have to collect coverage from inside the containers where Radius is running.

Reply:
I'm saying test coverage, not coverage results, I know coverage results are for unit tests. What I'm trying to understand is that why don't we have functional tests for these commands. Looks like a bunch of helper functions were added to be able to write functional tests but were never used.

Reply:
Ah sorry. No idea why then :)